.github/workflows/validate-pr.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: Missing `area/` label
         if: contains(join(github.event.pull_request.labels.*.name, ','), 'impact/') && !contains(join(github.event.pull_request.labels.*.name, ','), 'area/')
         run: |
-          echo "Every PR with an \`impact/*\` label should also have an \`area/*\` label"
+          echo "::error::Every PR with an \`impact/*\` label should also have an \`area/*\` label"
```

Comment:
heh.. same habit as I have; trying to use markdown everywhere 😂 - does this get rendered anywhere as actual markdown? Because if not, perhaps we should just use single-quotes.

Reply:
Nope: https://github.com/moby/moby/actions/runs/8570120714/job/23487391946?pr=47682#step:2:6, can change to single quote